src/editor/scene/scene-factory.ts

```diff
@@ -244,6 +251,13 @@ export default class SceneFactory {
 
         return mesh;
     }
+	
+	public static CreateCylinder (editor: Editor): Mesh {
+        const mesh = Mesh.CreateCylinder('New Cylinder', 1, 1, 1, 16, editor.core.scene);
+        this.AddToGraph(editor, mesh);
```

Comment:
Same, please add tags:
Tags.AddTagsTo(mesh, 'added');

Reply:
I don't understand what you mean by this

Reply:
You are right it is not needed, sorry :)